### PR TITLE
UI design

### PR DESCRIPTION
**Pull request by richardseo12**

### CODE REVIEW

File: Journey/Pages/Index.cshtml [hunk 1]
```diff
@@ -48,25 +112,30 @@
...
+          @* <p>@ViewData["SearchResults"]</p> *@
+          @using System.Web;
+          <p>@Html.Raw(((string)ViewData["SearchResults"]).Replace("\n", "<br />"))</p>
         </div>
-    </form>
+      </form>
+
+        <img id="imgReward" src="@ViewData["Reward"]" alt="Reward" />
+    </div>
+=======
```
Comment (jennifermarsman) [inline Journey/Pages/Index.cshtml:139]: Looks like merge formatting here that may need to be deleted?

File: Journey/Pages/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -4,12 +4,13 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"] - Journey</title>
-    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+@* Multiple stylesheets make it really difficult to debug. I'm only using the Journey.style.css. Feel free to re-enable the rest if needed
+ *@@*     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" /> *@
+@*     <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" /> *@
     <link rel="stylesheet" href="~/Journey.styles.css" asp-append-version="true" />
```
Comment (jennifermarsman) [inline Journey/Pages/Shared/_Layout.cshtml:10]: Did you remember to git add the Journey.styles.css file?

File: Journey/Pages/Index.cshtml (hunk 1 above)
Comment (richardseo12) [reply Journey/Pages/Index.cshtml:139]: oops, yes, this is my mistake

File: Journey/Pages/Shared/_Layout.cshtml (hunk 2 above)
Comment (richardseo12) [reply Journey/Pages/Shared/_Layout.cshtml:10]: I only edited the index.cshtml file. I guess we can take out all the css files

**Pull request merged**